packages/components/src/components/flyout/flyout-styles.ts

```diff
@@ -77,9 +76,12 @@ export const getComponentCss = (
       background: backgroundColor,
       opacity: isOpen ? 1 : 0,
       transform: isOpen ? 'initial' : `translate3d(${isPositionStart ? '-100%' : '100%'}, 0, 0)`,
-      transition: `opacity ${flyoutTransitionDuration} ${flyoutTransitionTimingFunction} ${
-        isOpen ? '0s' : flyoutTransitionDuration
-      }, transform ${flyoutTransitionDuration} ${flyoutTransitionTimingFunction}`,
+      transition: `${getTransition(
+        'opacity',
+        isOpen ? 'long' : 'short',
+        easing,
+        isOpen ? '0s' : 'long'
```

Comment:
There are like 4 `isOpen` checks for 3 properties, perhaps this entire part should become conditional.